python/kserve/kserve/storage.py

```diff
@@ -51,6 +51,7 @@
 class Storage(object):  # pylint: disable=too-few-public-methods
     @staticmethod
     def download(uri: str, out_dir: str = None) -> str:
+        uri = Storage._update_with_storage_spec(uri)
```

Comment:
hmm will this break user if someone is calling this API directly to download model?

Reply:
nvm looks like it returns back the uri if `STORAGE_CONFIG` or `STORAGE_OVERRIDE_CONFIG` is not specified

Reply:
Yes that's correct. It only updates the uri if STORAGE_CONFIG is specified, so there's no impact for the current storageURI behavior.